README.md

```diff
@@ -134,20 +134,33 @@ It will generate a `i18n-populator.config.json` file that you can edit later in
 ```json
 {
   "basePath": "example",
-  "translationEngines": ["google", "bing", "libreTranslate", "openAI", "deepL"],
-  "languages": [
+  "translationEngines": [
     {
-      "name": "en",
-      "files": ["en.json"]
+      "name": "google"
     },
     {
-      "name": "es",
-      "files": ["es.json"]
+      "name": "bing"
+    },
+    {
+      "name": "libreTranslate",
+      "mirrors": ["http://127.0.0.1:5000", "http://localhost:5001"]
     }
   ]
 }
 ```
 
+- 📌 Note: You can define one or more mirrors for LibreTranslate. The system will automatically select one available at the time of performing the translations.
+
+#### 🔍 What is a mirror?
+
+A mirror is an alternative instance of a service (in this case, LibreTranslate). By including multiple mirrors, you can ensure:
+
+- Reduced response time (the system selects the fastest or available one)
+
+- Redundancy in case of failures
+
+- Better load distribution if you manage multiple projects or users
+
```

Comment:
By the moment let's not include this information in the README, I think we should make changes on the wizard and the help menu to make this information to be available on the CLI program more than on the README of the project. And info like this (too specific of a single engine) can be added (on a future) on the `docs/` folder

So by the moment I think we can remove it. What do you think?